counter/src/Frame/Frame.js

```diff
@@ -0,0 +1 @@
+exports.view_ = require("~/src/Frame/Frame.jsx").Main
```

Comment:
Curious: Why not use a relative path here, i.e. `./Frame.jsx`? Is that a limitation of using `pulp` + `parcel` without Webpack?

Reply:
This is due to how purescript FFI works: the compiler just copies the FFI module to `output/` without modifying the paths, and therefore the relative path I'd have to use would be `../../src/Frame/Frame.jsx`. 

This has nothing to do with parcel or webpack. We have the same problem in MyCV too.